src/DevBetterWeb.Infrastructure/Data/Migrations/20210209190217_InviteEntity.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace DevBetterWeb.Infrastructure.Data.Migrations
+{
+    public partial class InviteEntity : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Invitations",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "int", nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
+                    Email = table.Column<string>(type: "nvarchar(max)", nullable: true),
+                    InviteCode = table.Column<string>(type: "nvarchar(max)", nullable: true),
+                    StripeSubscriptionId = table.Column<string>(type: "nvarchar(max)", nullable: true)
```

Comment:
Add an InviteConfig file in /Data/Config that sets the MaxLength on the string properties to something reasonable. There may be constants available for common things like Email.